public/index.html

```diff
@@ -2,36 +2,81 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="stylesheet" href="./styles.css">
```

Comment:
Тут лучше оставить %PUBLIC_URL% и сделать так:
```suggestion
    <link rel="stylesheet" href="%PUBLIC_URL%/styles.css">
```
```suggestion
    <link rel="stylesheet" href="./styles.css">
```